react/features/base/dialog/actionTypes.js

```diff
@@ -0,0 +1,22 @@
+import { Symbol } from '../react';
+
+/**
+ * The type of Redux action which begins a request to open a dialog.
+ *
+ * {
+ *     type: BEGIN_DIALOG_REQUEST,
+ *     component: React.Component,
+ *     props: React.PropTypes.object
+ * }
+ *
+ */
+export const BEGIN_DIALOG_REQUEST = Symbol('BEGIN_DIALOG_REQUEST');
```

Comment:
@saghul I was using the names that were used in native, what do you think about it? What about renaming the action types to SHOW_DIALOG and HIDE_DIALOG, wdyt?